recipe/meta.yaml

```diff
@@ -8,10 +8,10 @@ package:
 source:
     fn: Simbody-{{ version }}.tar.gz
     url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
-    sha256: 449c36e574d6f859d4fa8854ab6bc8e402e5ca5894bcce3e9fdce2f5658d64de
+    sha256: 7716d6ea20b950e71e8535faa4353ac89716c03fd7a445dd802eb6a630796639
 
 build:
-    number: 206
+    number: 0
```

Comment:
This may need to be a special number since it depends on open blas. I've been seeing 1000 for new conda 3 builds. Not quite sure why that is though.

Reply:
Oh interesting... thanks.